### ISSUE TITLE
Return Number object and number value randomly

### ISSUE THREAD

**Issue by kainstar** [color-js/color.js#583]
When I access some number properties like alpha, i will get random one from number value and Number object. 

```js
import Color from "colorjs.io";

new Color('red').to("srgb").alpha // number 1
new Color('rgba(255, 168, 233, .5)').to("srgb").alpha // Number object 0.5
```

It will cause some `typeof` compare get error result: 

```js
typeof (new Color('red').to("srgb").alpha) === 'number' // true
typeof (new Color('rgba(255, 168, 233, .5)').to("srgb").alpha)  === 'number' // false
```

Repureduce：
https://codesandbox.io/p/devbox/colorjs-io-demo-r2fv2d

Can we return number value in all scenarios, which is more compatible with ts type annotations?

**Comment by LeaVerou** [comment]
This is fixed in v0.6.0, which should be released soon.

**Comment by LeaVerou** [comment]
This is fixed in v0.6.0, which should be released soon, so I’ll close this. Please try it out and if you're still facing issues, please comment here and I'll reopen!
